bin/disco_vpc_ui.py

```diff
@@ -8,7 +8,10 @@
 import argparse
 import sys
 
-from disco_aws_automation import DiscoVPC
+from disco_aws_automation import (
+    DiscoVPC,
+    DiscoVPCPeerings
```

Comment:
The intention here was really breaking up the logic inside the original disco_vpc.py into separate smaller modules. We still include processing the VPC peerings in VPC life cycle, as a separate component.